cmake/std/atdm/ats2/environment.sh

```diff
@@ -242,7 +242,11 @@ export MPICC=`which mpicc`
 export MPICXX=`which mpicxx`
 export MPIF90=`which mpif90`
 
-export ATDM_CONFIG_MPI_POST_FLAGS="-disable_gpu_hooks;-map-by;socket:PE=4"
+export ATDM_CONFIG_MPI_EXEC=${ATDM_SCRIPT_DIR}/ats2/trilinos_jsrun
+
+# TODO: Need -np support from the trilinos_jsrun wrapper.
+export ATDM_CONFIG_MPI_POST_FLAGS="--rs_per_socket 4"
+export ATDM_CONFIG_MPI_EXEC_NUMPROCS_FLAG="-p"
```

Comment:
To be safe, we should add `unset ATDM_CONFIG_MPI_EXEC_NUMPROCS_FLAG` to:

* Trilinos/cmake/std/atdm/utils/unset_atdm_config_vars_environment.sh